src/modules/services/mysql.nix

```diff
@@ -23,16 +27,16 @@ let
     PATH="${lib.makeBinPath [ cfg.package pkgs.coreutils ]}:$PATH"
     set -euo pipefail
 
-    while ! ${cfg.package}/bin/mysqladmin ping ${optionalString (!isMariaDB) "-u root --password=''"} --silent; do
+    while ! MYSQL_PWD="" ${cfg.package}/bin/mysqladmin ping  -u root --silent; do
```

Comment:
could we refactor this as a global (as in current shell) function so everyone can reuse this?

Reply:
I guess that won't work so easy. Maybe exposing it somehow with nix?